web/styles/settings.css

```diff
@@ -834,24 +854,6 @@ input[type="checkbox"] {
 #organization-settings {
     .dropdown-widget-button {
         color: hsl(0deg 0% 33%);
```

Comment:
This color being individually written the same for multiple things, with no `light-dark` color, seems like a code smell. Should this property just be on `dropdown-widget-button` or this CSS deleted?